### PR TITLE
docs: updated table panel visualization

### PR DESCRIPTION
**Pull request by srperf**
**What is this feature?**

Updates to the table panel docs to include:

- modified a bit the description and reordered the beginning
- added a section mentioning the behavior when multiple data sets are present and added an image
- added a note on how to define max and minimums when using gauges as note. Looking forward for feedback on the best way to present this
- Transformed a section on Annotations and Alerts into a note. Looking forward to feedback on this change.

**Why do we need this feature?**

These changes will help users to understand better tables, some features that were not explained and flow better modifying some format.

**Who is this feature for?**

All Grafana users looking for docs on Table visualizations

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 1]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
 ## Annotation and alert support
 
-Annotations and alerts are not currently supported in tables.
+{{% admonition type="note" %}}
+At the moment, annotations and alerts are not supported by tables.
+{{% /admonition %}}
+
+## Table visualization functions
+
+The table visualization itself provides multiple functions.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:85]:
```suggestion
The table visualization provides multiple functions.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 2]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
-## Sort column
+### Sort column
 
 Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.
 
 ![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')
 
+### Data set selector
+
+The table panel visualization will display a dropdown list at the bottom if the data queried contains multiple data sets.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:95]:
```suggestion
If the data queried contains multiple data sets, a table displays a drop-down list at the bottom, so you can select the data set you want to visualize.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 3]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
+
+![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png 'Table Panel with multiple data sets')
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:97]: Only alt text is required. FYI, please take a look at the [image naming guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#media-asset-file-naming-conventions) for future reference. But don't worry about updating it for this PR.
```suggestion
![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 4]
```diff
@@ -157,6 +175,10 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 Cells can be displayed as a graphical gauge, with several different presentation types.
 
+{{% admonition type="note" %}}
+The maximum and minimum values of the Gauges will be configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:179]:
```suggestion
The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
```

**Review by imatwawana — COMMENTED** [review]
Thanks for these additions! For now I've focused my review on wording, but I'm going to do a structural review when I've had some more time to think.

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 5]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
 
 {{< docs/play title="Table Visualizations in Grafana" url="https://play.grafana.org/d/OhR1ID6Mk/" >}}
 
+### Key Features:
+- **Multi-Mode Support**: Seamlessly visualize time series data, detailed tables, annotations, or raw JSON inputs.
+- **Advanced Formatting**: Customize your data presentation with options for date formatting and value formatting.
+- **Coloring Options**: Enhance readability and insights by applying custom coloring schemes to your data points.
+- **Diverse Cell Types**: Elevate your data display with a variety of cell types, including: Colored text and backgrounds, Gauges, JSON, Images, Sparklines, and multiple combinations.
```
[the remark this answers is not in the document]
Comment (srperf) [reply docs/sources/panels-visualizations/visualizations/table/index.md:75]: Agreed, I was unsure of adding this part myself and now more that you pint out the marketing sound 😛 
If it doesn't make much sense we could remove it completely.

[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/table/index.md:75]: Yeah, giving it more thought, I don't think this fits here. Let's take it out.
```suggestion
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 6]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
+
 ## Annotation and alert support
 
-Annotations and alerts are not currently supported in tables.
+{{% admonition type="note" %}}
+Annotations and alerts are not currently supported for tables.
+{{% /admonition %}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:81]: A heading must be followed by some body text. I'm okay with making this a note, but it should either be a note without a heading or plain text with a heading. Do you think people need this information enough to keep a heading? If not, then we should remove the heading and leave it as a note. Also just updating the format of the admonition.
```suggestion
{{< admonition type="note" >}}
Annotations and alerts are not currently supported for tables.
{{< /admonition >}}
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 7]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
+
+## Table visualization functions
+
+The table visualization provides multiple functions.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:85]: I'd like to not add this heading as my plan is to eventually move all the configuration options under a new H2 to group them better. That will make all of these other tasks and features a bit more prominent.
```suggestion
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 8]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
 
-## Sort column
+### Sort column
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:87]:
```suggestion
## Sort column
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 9]
```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
...
 
 Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.
 
 ![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')
 
+### Data set selector
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:93]:
```suggestion
## Data set selector
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 10]
```diff
@@ -157,6 +175,10 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 Cells can be displayed as a graphical gauge, with several different presentation types.
 
+{{% admonition type="note" %}}
+The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
+{{% /admonition %}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:180]:
```suggestion
{{< admonition type="note" >}}
The maximum and minimum values of the gauges are configured automatically from the smallest and largest values in your whole data set. If you don't want the max/min values to be pulled from the whole data set, you can configure them for each column with field overrides.
{{< /admonition >}}
```

**Review by imatwawana — CHANGES_REQUESTED** [review]
I've reviewed the structure of the page and made some further suggestions for edits.

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 11]
```diff
@@ -70,14 +70,22 @@ The following video provides a visual walkthrough of the options you can set in
 
 ## Annotation and alert support
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:72]:
```suggestion
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 12]
```diff
@@ -68,16 +68,22 @@ The following video provides a visual walkthrough of the options you can set in
...
 
 Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.
 
 ![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')
 
+## Data set selector
+
+If the data queried contains multiple data sets, a table displays a drop-down list at the bottom, so you can select the data set you want to visualize.
+
+![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:85]:
```suggestion
![Table visualization with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```

**Review by imatwawana — APPROVED** [review]

**Comment by imatwawana** [comment]
Thanks for making those changes!

**Pull request merged**